src/graphql.tsx

```diff
@@ -635,7 +635,10 @@ class ObservableQueryRecycler {
   public recycle (observableQuery: ObservableQuery<any>): void {
     // Stop the query from polling when we recycle. Polling may resume when we
     // reuse it and call `setOptions`.
-    observableQuery.stopPolling();
+    observableQuery.setOptions({
+      fetchPolicy: 'cache-only',
```

Comment:
Let’s try instead saving the old options in `recycle()` and then merging those options with the new options in `reuse()`. Do you think that would work?

But perhaps this isn’t the right thing to do at all. Perhaps what we really need to do is create some kind of light-weight subscribe in `apollo-client` that will only allow receiving `updateQueries` and `refetchQueries` instead of writing workarounds in the recycler…